src/celeritas/io/ImportOpticalMaterial.hh

```diff
@@ -23,7 +23,7 @@ namespace celeritas
  */
 struct ImportScintComponent
 {
-    double yield_per_energy{};  //!< Yield for this material component [1/MeV]
+    double yield_frac{};  //!< Fraction of total scintillation yield
```

Comment:
I am pondering about the renaming here, since this has the same meaning as in `MatScintSpectrumRecord::yield_per_energy`, right? I.e. both are in [1/MeV], and are not a fraction of the total yield (e.g. the total is 200 photons per MeV for the material, and this component is 30% of that).

Reply:
We don't use these to calculate the total material yield in the import vector... if we got rid of that it would make sense to use non normalized values for these.

Reply:
I originally was going to check that the yields summed to the total material yield, but the couple of examples I saw, it was just fractional.